specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/apimskus.json

```diff
@@ -0,0 +1,364 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "SKUs available for Microsoft.ApiManagement",
+    "version": "2020-06-01-preview"
```

Comment:
>2020-06-01-preview [](start = 16, length = 18)

what's the reason to not use a more recent api version?

Reply:
This is our latest api version